lib/wasm/Makefile

```diff
@@ -11,8 +11,16 @@ init:
 clippy:
 	$(CLANG_PREFIX) cargo clippy --target=wasm32-unknown-unknown -- -D warnings
 
-pack:
```

Comment:
Seems we should change the instructions in the readme to use `make build `instead of` make pack`

Reply:
Fixed [6fba22f](https://github.com/breez/breez-sdk-liquid/pull/814/commits/6fba22f9f16597117d7a535b857bcb5156fb98e1)